api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -1770,7 +1770,7 @@ public void testCloneWorkspaceWithCohortsAndConceptSets() {
     Map<String, Cohort> cohortsByName = Maps.uniqueIndex(cohorts, c -> c.getName());
     Map<String, CohortReview> cohortReviewsByName =
         Maps.uniqueIndex(cohortReviews, c -> c.getCohortName());
-    assertThat(cohortsByName.keySet().size()).isEqualTo(2);
+    assertThat(cohortsByName.size()).isEqualTo(2);
```

Comment:
optional!  we can one step further and check `hasSize()`

Reply:
I like it! Thanks!